blocks.js

```diff
@@ -2599,7 +2599,7 @@ BlockMorph.prototype.restoreInputs = function (oldInputs) {
 
 BlockMorph.prototype.showHelp = function () {
     var myself = this,
-        ide = this.parentThatIsA(IDE_Morph),
+        ide = this.parentThatIsA(BlockEditorMorph).siblings()[0],
```

Comment:
Oh wait, that can't work, right? What if the block is in the regular scripting area, then it won't find the IDE anymore. Also, I'm a little nervous about assuming the IDE to always be the first sibling of a BlockEditor, although, I guess, it usually should be.

How about something like this (I didn't test this, just making this up now as I write this):

```JavaScript
var ide = this.parentThatIsA(IDE_Morph),
    blockEditor;
if (!ide) {
    blockEditor = this.parentThatIsA(BlockEditorMorph);
    if (blockEditor) {
        ide = blockEditor.target.parentThatIsA(IDE_Morph);
    }
}
````